src/main/java/com/bravo/user/model/dto/UserSaveDto.java

```diff
@@ -8,4 +8,23 @@
   private String firstName;
   private String middleName;
   private String lastName;
+  private Integer phoneNumber;
+
+  public String getFirstName() {
```

Comment:
These getters/setters are not necessary since the [Lombok](https://projectlombok.org/) `@Data` annotation provides them.